common/lib/capa/capa/templates/crystallography.html

```diff
@@ -1,3 +1,4 @@
+<%! from openedx.core.djangolib.markup import HTML %>
 <section id="inputtype_${id}" class="capa_inputtype" >
```

Comment:
I wonder what it is!! I agree, let's delete the related code in a follow up story.